packages/client/src/components/Modals/SavedSearchPanel.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div>
+    <b-button v-b-toggle.saved-search-panel variant="outline-secondary">
+      My Saved Searches
+    </b-button>
+    <b-sidebar
+      id="saved-search-panel"
+      class="saved-search-panel"
+      right
+      shadow
+    >
+    <template #header="{ hide }">
+      <div class="saved-search-title">Saved Searches</div>
+      <div @click="hide">
+        <svg viewBox="0 0 16 16" width="1em" height="1em" focusable="false" role="img" aria-label="x" xmlns="http://www.w3.org/2000/svg" fill="currentColor" class="bi-x b-icon bi header-close-button"><g><path d="M4.646 4.646a.5.5 0 0 1 .708 0L8 7.293l2.646-2.647a.5.5 0 0 1 .708.708L8.707 8l2.647 2.646a.5.5 0 0 1-.708.708L8 8.707l-2.646 2.647a.5.5 0 0 1-.708-.708L7.293 8 4.646 5.354a.5.5 0 0 1 0-.708z"></path></g></svg>
+      </div>
```

Comment:
Hey I think if you do this it might be a bit shorter than having to use the entire svg element. 
```
<b-button type="button" class="close" aria-label="Close" @click="hide">
  <span aria-hidden="true">&times;</span>
</b-button>
```